docs/diagrams/CommandClassDiagram.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor LOGIC_COLOR_T4
+skinparam classBackgroundColor LOGIC_COLOR
+
+package Command {
+
+Class "{abstract}\nCommand" as Command
+Class XYZCommand
+Class "{abstract}\nABCCommand" as ABCCommand
+Class XYZInternshipCommand
+Class XYZSkillCommand
+Class XYZResumeCommand
+Class XYZProjectCommand
+
+}
+
+XYZInternshipCommand -up-|> ABCCommand
+XYZSkillCommand -up-|> ABCCommand
+XYZResumeCommand -up-|> ABCCommand
+XYZProjectCommand -up-|> ABCCommand
```

Comment:
Would it make more sense to write `ABCResumeCommand -up-|> ABCCommand`, instead of `XYZResumeCommand`?

Reply:
Shit yes, I'm blind hahahaa